portal/src/main/webapp/js/src/data_download.js

```diff
@@ -200,6 +210,70 @@ var DataDownloadTab = (function() {
         $("#text_area_case_affected").append(strs.case_affected);
         $("#text_area_case_matrix").append(strs.case_matrix);
     }
+    //TODO : include study name(id) incase of sample id match across studies?
```

Comment:
@kalletlak we would need to include the study names/ids in any places of sample ids.